src/logistration/tests/LoginPage.test.jsx

```diff
@@ -219,4 +219,62 @@ describe('LoginPage', () => {
     const loginPage = mount(reduxWrapper(<IntlLoginPage {...props} />));
     expect(loginPage.find('#tpa-alert').find('span').text()).toEqual(expectedMessage);
   });
+
+  it('should display institution login button', () => {
+    store = mockStore({
+      ...store,
+      logistration: {
+        ...store.logistration,
+        loginResult: {
+          success: true,
+          redirectUrl: '',
+        },
+        thirdPartyAuthContext: {
+          providers: [],
+          secondaryProviders: [
+            {
+              id: 'saml-test',
+              name: 'Test University',
+              loginUrl: '/dummy-auth',
+              registerUrl: '/dummy_auth',
+            },
```

Comment:
Can we define this as a const outside of test? This way we won't have to write repeated code for all tests that require the secondaryProviders